tests/test_itisatrap_page.py

```diff
@@ -1,13 +1,13 @@
-import os
 import pytest
 from pages.itisatrap import ItisaTrapPage
 
 
-#PREF_SET = os.environ['PREF_SET']
+# PREF_SET = os.environ['PREF_SET']
```

Comment:
pref_set is now being passed on as a fixture? I believe.  in any event, this line seems no longer needed so we can remove